### PR TITLE
Fix invalid API call when running locally

### PR DESCRIPTION
**Pull request by simon-engledew**
The method `getAnalysisKey` would call `getWorkflowPath` and raise an API Error.

This change follows the pattern in `prepareLocalRunEnvironment` to set a dummy value for the required environment variable, therefore shortcutting the API request.

I have tried to mirror the existing mechanism for stubbing out `GITHUB_JOB` as closely as possible.

### Merge / deployment checklist

- [x] Confirm this change is backwards compatible with existing workflows.
- [x] Confirm the [readme](https://github.com/github/codeql-action/blob/master/README.md) has been updated if necessary.

### CODE REVIEW

File: src/actions-util.test.ts [hunk 1]
```diff
@@ -33,30 +33,59 @@ test("getRef() returns head PR ref if GITHUB_SHA not currently checked out", asy
...
+  process.env.CODEQL_ACTION_ANALYSIS_KEY = "";
+  process.env.GITHUB_JOB = "";
+
+  actionsutil.prepareLocalRunEnvironment();
+
+  const actualAnalysisKey = await actionsutil.getAnalysisKey();
+
+  t.deepEqual(actualAnalysisKey, "LOCAL-RUN:UNKNOWN-JOB");
+
+  process.env.CODEQL_LOCAL_RUN = origLocalRun;
```
Comment (robertbrignull) [inline src/actions-util.test.ts:49]: The env is reset between tests, so setting this back shouldn't be necessary. It's a bit hacky but look in `testing-utils.ts` for the implementation.

Although I realise the test below this also has this pattern. It should be safe to remove `origLocalRun` from both of these tests.

**Review by robertbrignull — COMMENTED** [review]
Looks reasonable. So just to confirm, was it previously failing with a "Invalid API call in local run" error?

**Comment by simon-engledew** [comment]
> Looks reasonable. So just to confirm, was it previously failing with a "Invalid API call in local run" error?

Yes, exactly! 👍 Any attempts to access the API when local run is turned on automatically raise an Error.

File: src/actions-util.test.ts (hunk 1 above)
Comment (simon-engledew) [reply src/actions-util.test.ts]: Ah nice! 🧹  I have removed the unnecessary lines.

**Review by robertbrignull — APPROVED** [review]
LGTM

**Pull request merged**